frontend/src/Components/Table/Table.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+
+import { DataItem, Categories } from "../../Utilities/dataUtilities";
+
+import styles from "./Table.module.scss";
+
+interface Props {
+  data: Array<DataItem>;
+  categories: Array<Categories>;
+  identity: boolean;
+}
+
+const Table = (props: Props) => {
+  // regular table
+  const headers = props.categories.map((category) => (
+    <th className={styles.headerField}>{category.title}</th>
+  ));
+
+  let rows = props.data
+    .map((item: DataItem | any) => {
+      return (
+        <tr className={styles.dataRows}>
+          {props.categories.map((category: Categories) => {
+            return <td className={styles.dataField}>{item[category.field]}</td>;
+          })}
+        </tr>
+      );
+    })
+    .slice(0, 8);
+
+  //identity table to accomodate odd data structure of identity product
+  const identittyHeaders = props.categories.map((category) => (
+    <span className={styles.identityHeader}>{category.title}</span>
+  ));
+
+  const idendityRows = props.data.map((item: DataItem | any) => {
```

Comment:
(nit) `idendityRows` and `identittyHeaders ` are misspelled

Reply:
done